python/feathub/feature_views/sliding_feature_view.py

```diff
@@ -270,6 +271,50 @@ def build(
         )
         return feature_view
 
+    def get_step_size(self) -> timedelta:
+        """
+        Returns the step size of the sliding window features in the feature view. All
+        the sliding window features must have the same step size.
```

Comment:
It has been recorded in SlidingFeatureView's python doc that "SlidingWindowTransform used in this feature view must have the same step size and group-by keys", so it might be unnecessary to mention this requirement again here.